Update zklogin.mdx with recommended machine specs (#16748)
MystenLabs/sui: docs/content/concepts/cryptography/zklogin.mdx

   - `/ping`: To test if the service is up. Running `curl http://localhost:PROVER_PORT/ping` should return `pong`.
   - `/v1`: The request and response are the same as the Mysten Labs maintained service.

A few important things to note:

> - The backend service (mysten/zklogin:prover-stable) is compute-heavy. We recommend using at least 16 cores and 16GB RAM. Using weaker instances will likely lead to timeout errors with the message "Call to rapidsnark service took longer than 15s". Note that the environment variable `PROVER_TIMEOUT` can be adjusted to set a different timeout value, e.g., `PROVER_TIMEOUT=30` for a timeout of 30 seconds.

Reviewer: ```suggestion
- The backend service (mysten/zklogin:prover-stable) is compute-heavy. Use at least the minimum recommended 16 cores and 16GB RAM. Using weaker instances can lead to timeout errors with the message "Call to rapidsnark service took longer than 15s". You can adjust the environment variable `PROVER_TIMEOUT` to set a different timeout value, for example, `PROVER_TIMEOUT=30` for a timeout of 30 seconds.
```
